Makefile

```diff
@@ -604,6 +604,20 @@ kurl-util-image: ## Download Kurl util image (replicated/kurl-util:alpha)
 watchrsync: ## Syncronize the code with a remote server. More info: CONTRIBUTING.md
 	bin/watchrsync.js
 
+.PHONY: build/sample
+build/sample: ## Build sample config under testdata
+	cp ./testdata/sample/Manifest ./scripts/Manifest
+	make clean
+	make build/packages/kubernetes/1.25.3/ubuntu-22.04
```

Comment:
this will only work on ubuntu 22 correct? should you mention that in the docs?

Reply:
Good point.

Reply:
@emosbaugh wdyt now?